python/pylibcugraph/pyproject.toml

```diff
@@ -41,14 +41,12 @@ classifiers = [
 [project.optional-dependencies]
 test = [
     "cudf==23.10.*",
-    "networkx>=2.5.1",
     "numpy>=1.21",
     "pandas",
     "pytest",
     "pytest-benchmark",
     "pytest-cov",
     "pytest-xdist",
-    "scipy",
```

Comment:
I need to double check if it's safe to drop nx and scipy from the PLC test deps.

Reply:
`scipy` is needed, it was added back.